env/WCOSS2.env

```
@@ -85,7 +85,17 @@ elif [[ "${step}" = "snowanl" ]]; then
     [[ ${NTHREADS_SNOWANL} -gt ${nth_max} ]] && export NTHREADS_SNOWANL=${nth_max}
     export APRUN_SNOWANL="${launcher} -n ${npe_snowanl}"
 
-    export APRUN_APPLY_INCR="${launcher} -n 6"
+    export APRUN_APPLY_INCR="${launcher} -n ${npe_apply_incr}6"
```

```suggestion```
    export APRUN_APPLY_INCR="${launcher} -n ${npe_apply_incr}
```